src/components/MenuXss.vue

```diff
@@ -117,6 +117,10 @@
           />
         </VList>
       </VMenu>
+      <VListItem
+        title="atob() helper"
+        @click="applyFunction('Encode.AtobHelper.encode')"
+      />
```

Comment:
Could you move this after `CTF Snippets`?

I would like to group pure list item together.

Reply:
Sure